sources/org.osbuild.ostree

```diff
@@ -74,18 +74,13 @@ def ostree(*args, _input=None, **kwargs):
                    check=True)
 
 
-def download(items, cache):
-    # Prepare the cache and the output repo
-    repo_cache = os.path.join(cache, "repo")
-    ostree("init", mode="archive", repo=repo_cache)
-
-    # Make sure the cache repository uses locks to protect the metadata during
-    # shared access. This is the default since `2018.5`, but lets document this
-    # explicitly here.
-    ostree("config", "set", "repo.locking", "true", repo=repo_cache)
-
-    for commit, item in items.items():
-        remote = item["remote"]
+class OSTreeSource(sources.SourceService):
+
+    content_type = "org.osbuild.ostree"
+
+    # pylint: disable=arguments-renamed
+    def fetch_one(self, commit, desc):
```

Comment:
`checksum` instead of commit is ok, I think,  and then we could have a `commit = checksum` to avoid renaming all the code though.